testing/demo/pkg/setup/groth16.go

```diff
@@ -182,3 +177,14 @@ func getEvmProverInfo() (*proverclient.InfoResponse, error) {
 	}
 	return info, nil
 }
+
+func getGroth16Vk() ([]byte, error) {
+	dir, err := os.Getwd()
+	buf := bytes.NewBuffer(nil)
+	vkFile, err := os.Open(dir + "/ibc/lightclients/groth16/groth16_vk.bin")
```

Comment:
is this file guaranteed to exist or is it generated by some command? Is there anyway to hard-code the verifier key so that we don't have to rely on reading a file on disc?